shoes-swt/lib/shoes/swt/input_box.rb

```diff
@@ -32,7 +32,7 @@ def text=(value)
       end
 
       def enabled(value)
-        @real.enable_widget value
+        @real.enabled = value
```

Comment:
this change is a bit unrelated, I couldn't find any reference to `enableWidget` in the SWT docs so I changed it to the documented enabled